wave_kinematics/include/wave/kinematics/rotation.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef __WAVE_KINEMATICS_ROTATION_HPP__
+#define __WAVE_KINEMATICS_ROTATION_HPP__
+
+#include <iostream>
+#include <functional>
+
+#include "wave/utils/utils.hpp"
+#include <kindr/Core>
+
+namespace wave {
+
+class Rotation {
+public:
+  // Internal storage as a kindr Rotation object.
+  kindr::RotationMatrixD rotation_object;
```

Comment:
The goal of the wrapper is to provide an interface that is independent of the underlying implementation.  If we one day decide to dump Kindr and move to Sophis, TooN, or even just Eigen, the change is encapsulated within the kinematics classes.  To the rest of our code, the change is transparent.  Is this still possible to do if we inherit from RotationMatrix, or would we be stuck with their member function names and signatures?